content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -29,45 +29,46 @@ The client must know two information about the server:
 1.IP address of Server and 
 2.Port number
 
->Port number 0-1024 are reserved for administrative purposes(e,g 21 for FTP,23 for Telnet,25 for email and 80 for HTTP).
+>Port number 0-1024 are reserved for administrative purposes(e,g 21 for FTP,23 for Telnet,25 for email and 80 for HTTP).In our program we will use port number 5000.
 
-This happens when a TCP connection is being used between two machines:-
+Whenever we create a socket connection in Java, the following steps take place:-
 
 1. server creates a ServerSocket object, to indicate which port number our communication is to occur. If an I/O  error occurs, exception handling mechanisms are implemented; **Throws** an **I/OException** or a **try** **catch** block is used.
```

Comment:
Change:

>server creates a ServerSocket object, to indicate which port number our communication is to occur. If an I/O error occurs, exception handling mechanisms are implemented; **Throws** an **I/OException** or a **try** **catch** block is used.

To:

**1. The Server creates a `ServerSocket` object, to indicate which port number our communication is to occur. If an I/O error occurs, exception handling mechanisms are triggered; **Throws** an **I/OException** or a **try** **catch** block is used.**

Small formatting change, I tried to make it so whenever you introduce a class name it you enclose it in ticks (\`). While not necessary you did do that a couple of times after I suggested it so I'm doing this for consistency. While I personally like doing this every time I mention a class or method name, as long as it's consistent it's fine. Also, added a *the* at the start of the step to be more grammatically correct, and changed *implemented* to *triggered* as a more precise word choice.